src/ucp/wireup/address.c

```diff
@@ -1420,6 +1456,24 @@ ucp_address_do_pack(ucp_worker_h worker, ucp_ep_h ep, void *buffer, size_t size,
         }
     }
 
+    if (ucp_address_pack_v1_extra_info(addr_version, pack_flags)) {
+        /* Pack our release version in a wire compatible way:
+         * pack a pseudo empty device after all real devices and specify
+         * release version instead of md index.
+         */
+        UCS_STATIC_ASSERT(UCP_RELEASE_CURRENT <
+                          UCP_ADDRESS_WIRE_VERSION_MASK_V1);
+        ucs_assertv(addr_version == UCP_OBJECT_VERSION_V1, "addr_version=%u",
+                    addr_version);
+        ptr = ucp_address_pack_md_info(ptr, 1, 0, UCP_RELEASE_CURRENT,
+                                       addr_version);
+        /* Mark this device as the last one with 0 length address */
+        *ucs_serialize_next(&ptr, uint8_t) = UCP_ADDRESS_FLAG_LAST;
+    } else if (num_devices > 0) {
+        ucs_assert(dev_flags_ptr != NULL);
+        *(uint8_t*)dev_flags_ptr |= UCP_ADDRESS_FLAG_LAST;
```

Comment:
do we still need line a452?

Reply:
yes, this is for iface or ep length fields